apidoc/Titanium/UI/AlertDialog.yml

```diff
@@ -168,6 +168,14 @@ properties:
         On iOS and Mobile Web, set to `-1` to disable the cancel option.
     type: Number
     default: undefined (Android), -1 (iOS, Mobile Web)
+
+  - name: destructive
+    summary: Index to define the destructive button.
+    platforms: [iphone, ipad]
+    description: |
+        Note that this property available on iOS SDK 8 or above. 
+    type: Number
+    default: -1
```

Comment:
Would be nice to specify that setting it to `-1` disables this option, or does nothing. Having a default value of `-1` without explaining what it does makes it unclear